astropy_bot/autolabel.py

```diff
@@ -0,0 +1,24 @@
+from os import path, sep
+from baldrick.plugins.github_pull_requests import pull_request_handler
+
+
+@pull_request_handler
+def autolabel_subpackages(pr_handler, repo_handler):
```

Comment:
If you want this to have on/off toggle, you can have it pull user setting not unlike this

https://github.com/astropy/astropy-bot/blob/dd2c34eaf69eabaf06f11283f2e4327cdef3d4f6/astropy_bot/changelog_checker.py#L16

Reply:
Where does this config setting get pulled from? I can't find an example in the astropy repo.

Reply:
It is pulled from a cunning place.

https://github.com/OpenAstronomy/baldrick/blob/b467c7d1e6044eac283489010550b1cb9e56fd78/baldrick/github/github_api.py#L129